core/services/pipeline/task.http.go

```diff
@@ -144,7 +144,11 @@ func (t *HTTPTask) Run(ctx context.Context, taskRun TaskRun, inputs []Result) Re
 		"response", string(responseBytes),
 		"url", t.URL.String(),
 	)
-	return Result{Value: responseBytes}
+	// NOTE: We always stringify the response since this is required for all current jobs.
```

Comment:
Why stringify?

Reply:
We spoke about this - because otherwise we save base64 from the http request which is not very helpful